lib/bridge.js

```diff
@@ -651,6 +692,15 @@ Bridge.prototype._onAliasQuery = function(alias) {
             }
             // store the matrix room only
             return self._roomStore.setMatrixRoom(matrixRoom);
+        }).then(function () {
+            // Process group-related functions, if we need to.
+            if (!dontAdvertiseGroup && self.groupId) {
+                return self._createGroupIfNotExists().then(function() {
+                    return self._botClient.sendStateEvent(roomId, "m.room.related_groups", {
```

Comment:
This won't handle the case where there are other groups associated here, but I guess this is a non-issue for portal rooms?

Reply:
It shouldn't be an issue, although there's a chance that a bridge may shove some other related groups in before we get here. Adding in the safety wouldn't hurt.